### PR TITLE
2nd batch of rename edits to Azure Cosmos DB for PostgreSQL

### PR DESCRIPTION
**Pull request by clairegiordano**

### CODE REVIEW

File: get_started/what_is_citus.rst [hunk 1]
```diff
@@ -3,34 +3,33 @@
...
 
-How Far Can Citus Scale?
-------------------------
+Citus Gives You Postgres At Any Scale
+-------------------------------------
 
-Citus scales horizontally by adding worker nodes, and vertically by upgrading workers/coordinator. In practice our customers have achieved the following scale, with room to grow even more:
+You can start using Citus on a single node, using a distributed data model from the beginning so you are "scale out ready". When your Postgres workload needs to scale, it's easy to add worker nodes to the Citus database cluster, and/or to scale up the coordinator and worker nodes in your cluster. 
+
+Sometimes people ask "how big can Citus scale"? Here are a few examples of large-scale customers—but please keep in mind that there are many, many 2-node and 3-node Citus clusters in the wild, too.
```
Comment (jonels-msft) [inline get_started/what_is_citus.rst:32]: I think suggested English usage is to put the question mark inside the quotation mark.

File: get_started/what_is_citus.rst [hunk 2]
```diff
@@ -44,13 +43,13 @@ Citus scales horizontally by adding worker nodes, and vertically by upgrading wo
...
   * 80B rows updated/day
-  * 20-node Citus database cluster on Google Cloud
+  * 20-node Citus database cluster
   * 2.4TB memory, 1280 cores, and 80TB of data
   * ...with plans to grow to 45 nodes
 
 * `MixRank <https://www.citusdata.com/customers/mixrank>`_
 
-  * 10+ PB of time series data
+  * 1.6 PB of time series data
```
Comment (jonels-msft) [inline get_started/what_is_citus.rst:52]: Oh, did this change back again? I recall increasing the number recently.

**Review by jonels-msft — CHANGES_REQUESTED** [review]
The edits make a lot of sense. I also didn't see any stray Hyperscale references on this branch, so that's good.

Just two small notes to address.

File: get_started/what_is_citus.rst (hunk 2 above)
Comment (clairegiordano) [reply get_started/what_is_citus.rst]: you're right, it's 10PB. fixed.

**Review by jonels-msft — APPROVED** [review]
Niccce

**Pull request merged**